### PR TITLE
Update service-new-look-where-actions.md

### PR DESCRIPTION
**Pull request by wim-dp**
Share and Export menu have changed in the new look

### CODE REVIEW

**Comment by PRMerger7** [comment]
@wim-dp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/consumer/service-new-look-where-actions.md [hunk 1]
```diff
@@ -36,16 +36,16 @@ Looking for information about the new look in **Power BI Desktop**? See [Use the
 | Actual size | View > Actual size | Menu bar View (right side) > Actual size |
 | High-contrast colors | View > High-contrast colors | Menu bar View (right side) > High-contrast colors |
 | Bookmarks pane | View > Bookmarks pane |  Menu bar Bookmarks > Show more bookmarks |
-| Refresh | Refresh | Menu bar More options (...) > Refresh |
+| Refresh | Refresh | Refresh |
```
Comment (maggiesMSFT) [inline powerbi-docs/consumer/service-new-look-where-actions.md:39]:
```suggestion
| Refresh | Refresh | Menu bar (right side) Refresh |
```

File: powerbi-docs/consumer/service-new-look-where-actions.md [hunk 2]
```diff
@@ -36,16 +36,16 @@ Looking for information about the new look in **Power BI Desktop**? See [Use the
 | Actual size | View > Actual size | Menu bar View (right side) > Actual size |
 | High-contrast colors | View > High-contrast colors | Menu bar View (right side) > High-contrast colors |
 | Bookmarks pane | View > Bookmarks pane |  Menu bar Bookmarks > Show more bookmarks |
```
Comment (maggiesMSFT) [inline powerbi-docs/consumer/service-new-look-where-actions.md:38]:
```suggestion
| Bookmarks pane | View > Bookmarks pane |  Menu bar Bookmarks (right side) > Show more bookmarks |
```

**Review by maggiesMSFT — CHANGES_REQUESTED** [review]
Thanks! a couple changes

**Comment by PRMerger17** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@maggiesMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**